src/ctffind_runner.cpp

```diff
@@ -713,7 +713,7 @@ void CtffindRunner::executeCtffind4(long int imic)
 
 	// Write script to run ctffind
 	fh << "#!/usr/bin/env " << fn_shell << std::endl;
-	fh << fn_ctffind_exe << ctffind4_options << " > " << fn_log << " << EOF"<<std::endl;
+	fh << fn_ctffind_exe << ctffind4_options << " > '" << fn_log << "' << EOF"<<std::endl;
```

Comment:
ver5.0-cryocloud branch also has support for Ctffind5, so I guess these changes should also be applied under `void CtffindRunner::executeCtffind5` and `CtffindRunner::getCtffind5Results`

Reply:
Rebased on ver5.0-cryocloud and updated also commands for ctffind5